docs/UserGuide.md

```diff
@@ -155,6 +182,20 @@ Format: `find KEYWORD [MORE_KEYWORDS]`
 * Only full words will be matched e.g. `Han` will not match `Hans`
 * Persons matching at least one keyword will be returned (i.e. `OR` search).
   e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
```

Comment:
Would you want to put a full stop here? Not sure since it is in code format. Minor issue 👍